### ISSUE TITLE
Not playing on Chrome

### ISSUE THREAD

**Issue by xenyal** [michaelbromley/soundcloud-visualizer#4]
Pasted the following link onto the input form (https://soundcloud.com/hotelgarudamusic/rhythm-of-the-night-hotel), clicked play and the audio player loads the associated song info / artwork into `#trackInfoPanel`, but the audio does not load.

**Comment by michaelbromley** [comment]
Yeah, I think it is related to this: https://github.com/michaelbromley/soundcloud-visualizer/issues/3#issuecomment-110434827

Basically SoundCloud's CDNs are not all configured correctly to allow cross-origin resource sharing. It's a known issue.

**Comment by surayashivji** [comment]
It works when I go to your Demo URL but not when I run it locally -- any idea why?

**Comment by michaelbromley** [comment]
When you are running it locally, are you running a local web server, or just opening the file in the browser?

**Comment by pedropcamellon** [comment]
I'm running a server. Exactly Live Server in Vs Code. I tried registering a new app because I meed client ID I guess but SoundCloud is not accepting new requests. Anyone can share its ID?
